ui/raidboss/data/02-arr/dungeon/aurum_vale.ts

```diff
@@ -0,0 +1,33 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheAurumVale,
+  triggers: [
+    {
+      id: 'Aurum Vale GoldLungs/Burrs',
+      type: 'GainsEffect',
+      // Count `0[2-9]` here is to filter out the first stack
+      netRegex: NetRegexes.gainsEffect({ effectId: ['12E', '12F'], count: '0[2-9]' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Eat Fruit',
+          de: '',
```

Comment:
In general, leave out lines that don't have translations.  Everything defaults to English if missing and "find missing translations" scripts will not find this if you leave a blank.